commit 90dc30b08a06a4ce06b53d1fd0dd096a8f19cd2f
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Fri Jan 11 21:40:40 2019 +0800

    fix: a crash in ImageWidget::event
    
    The function is bool typed and should return a correct value. This fixes
    the crash in https://github.com/linuxdeepin/developer-center/issues/609
    
    Change-Id: I2e869910ce186714a480a83a2aa6cc57511d85dd

diff --git a/viewer/module/view/scen/imageview.cpp b/viewer/module/view/scen/imageview.cpp
index ab08410..3419a60 100644
--- a/viewer/module/view/scen/imageview.cpp
+++ b/viewer/module/view/scen/imageview.cpp
@@ -496,7 +496,7 @@ bool ImageView::event(QEvent *event)
     if (event->type() == QEvent::Gesture)
         handleGestureEvent(static_cast<QGestureEvent*>(event));
 
-    QGraphicsView::event(event);
+    return QGraphicsView::event(event);
 }
 
 void ImageView::onCacheFinish()
